File: utils/appwrite_client.py
import os
from appwrite.client import Client
from appwrite.services.account import Account
from appwrite.services.databases import Databases
from appwrite.services.storage import Storage

client = Client()
client.set_endpoint(os.getenv('APPWRITE_ENDPOINT', 'https://cloud.appwrite.io/v1'))
client.set_project(os.getenv('APPWRITE_PROJECT_ID'))
client.set_key(os.getenv('APPWRITE_API_KEY'))

account = Account(client)
databases = Databases(client)
storage = Storage(client)

File: functions/auth/src/appwrite-client.js
const { Client } = require('node-appwrite');

function createAppwriteClient(req) {
    const client = new Client();
    client
        .setEndpoint(process.env.APPWRITE_ENDPOINT || 'https://cloud.appwrite.io/v1')
        .setProject(process.env.APPWRITE_PROJECT_ID)
        .setKey(req.variables['APPWRITE_API_KEY']);
    
    return client;
}

module.exports = { createAppwriteClient };